templates/theme/theme-ircc-en.html

```diff
@@ -0,0 +1,216 @@
+				</ul>
+			</div>
+		</section>
+		<section class="gc-most-requested">
+			<div class="container">
+				<h2>Most requested</h2>
+				<ul>
+					<li><a href="https://canada.ca/en/immigration-refugees-citizenship/services/application/check-status.html">Application status</a></li>
+					<li><a href="https://canada.ca/en/immigration-refugees-citizenship/services/application/check-processing-times.html">Processing times</a></li>
+				</ul>
+			</div>
+		</section>
+		<div class="container">
+			<section class="gc-srvinfo">
+				<h2 class="mt-0">IRCC application</h2>
+				<h3><a href="https://canada.ca/en/immigration-refugees-citizenship/services/application.html">Your IRCC application</a></h3>
+				<p>Accounts (GCKey, representatives, employers and more), biometrics, forms, change address</p>
+			</section>
+		</div>
+		<div class="bg-light">
+			<div class="container">
+				<section class="gc-srvinfo">
+					<h2>Visas, permits and immigration</h2>
+					<div class="wb-eqht-grd row">
+						<div class="col-md-6">
+							<h3><a href="https://canada.ca/en/immigration-refugees-citizenship/services/visit-canada.html">Visit Canada</a></h3>
+							<p>Visitor visa, electronic travel authorization (eTA), super visa, transit visa, visit on business, extend your visit</p>
+						</div>
+						<div class="col-md-6">
+							<h3><a href="https://canada.ca/en/immigration-refugees-citizenship/services/immigrate-canada.html">Live in Canada permanently</a></h3>
+							<p>Permanent residence (PR) programs such as Express Entry, family sponsorship and the Provincial Nominee Program (PNP)</p>
+						</div>
+						<div class="col-md-6">
+							<h3><a href="https://canada.ca/en/immigration-refugees-citizenship/services/work-canada.html">Work in Canada temporarily</a></h3>
+							<p>Work permits for up to 3 years, post-graduation work permit (PGWP), hire foreign workers</p>
+						</div>
+						<div class="col-md-6">
+							<h3><a href="https://canada.ca/en/immigration-refugees-citizenship/services/study-canada.html">Study in Canada</a></h3>
+							<p>Study permits, designated learning institutions (DLIs), work while you study, change schools</p>
+						</div>
+						<div class="col-md-6">
+							<h3><a href="https://canada.ca/en/immigration-refugees-citizenship/services/refugees.html">Refugees and asylum</a></h3>
+							<p>Claim asylum, resettle as a refugee, sponsor a refugee, Economic Mobility Pathways Pilot (EMPP)</p>
+						</div>
+						<div class="col-md-6">
+							<h3><a href="https://ircc.canada.ca/iec/index.asp">International Experience Canada (IEC)</a></h3>
+							<p>Special program for youth aged 18 to 35 to work and travel abroad, including working holiday visas</p>
+						</div>
+					</div>
+					<div class="well">
+						<h3>Not sure where to start?</h3>
+						<p><a href="#" class="stretched-link">Answer a few questions and explore all immigration programs</a></p>
+					</div>
```

Comment:
I am not sure how we will be able to get this implemented in mws. I do like the simplicity of the markup.

May be this should be an extension of our Service and Information component which do leverage a fix pattern inside. Let's discuss.